src/main/java/uk/gov/companieshouse/mapper/DissolutionResponseMapper.java

```diff
@@ -17,12 +26,39 @@ public DissolutionCreateResponse mapToDissolutionCreateResponse(Dissolution diss
         return response;
     }
 
-    private DissolutionCreateLinks generateLinks(String companyNumber) {
-        final DissolutionCreateLinks links = new DissolutionCreateLinks();
+    public DissolutionGetResponse mapToDissolutionGetResponse(Dissolution dissolution) {
+        final DissolutionGetResponse response = new DissolutionGetResponse();
+
+        response.setETag(dissolution.getData().getETag());
+        response.setKind(DISSOLUTION_KIND);
+        response.setLinks(generateLinks(dissolution.getCompany().getNumber()));
+        response.setApplicationStatus(dissolution.getData().getApplication().getStatus());
+        response.setApplicationReference(dissolution.getData().getApplication().getReference());
+        response.setApplicationType(dissolution.getData().getApplication().getType());
+        response.setCompanyName(dissolution.getCompany().getName());
+        response.setCompanyNumber(dissolution.getCompany().getNumber());
+        response.setCreatedAt(Timestamp.valueOf(dissolution.getCreatedBy().getDateTime()));
+        response.setCreatedBy(dissolution.getCreatedBy().getUserId());
+        response.setDirectors(mapToDissolutionGetDirectors(dissolution.getData().getDirectors()));
+        return response;
+    }
+
+    private DissolutionLinks generateLinks(String companyNumber) {
+        final DissolutionLinks links = new DissolutionLinks();
 
         links.setSelf(String.format("/dissolution-request/%s", companyNumber));
         links.setPayment(String.format("/dissolution-request/%s/payment", companyNumber));
 
         return links;
     }
+
+    public List<DissolutionGetDirector> mapToDissolutionGetDirectors(List<DissolutionDirector> directors) {
+        return directors.stream().map(dir -> {
+            DissolutionGetDirector getDirector = new DissolutionGetDirector();
+            getDirector.setName(dir.getName());
+            getDirector.setEmail(dir.getEmail());
```

Comment:
don't forget about mapping onBehalfName to the response if it is present in the db. Otherwise how will the UI know that someone has singed on behalf of this director?